docs/ftldns/configfile.md

```diff
@@ -311,6 +320,10 @@ Log information regarding FTL's embedded event handling queue.
 
 Log information about script helpers, e.g., due to `dhcp-script`.
 
+#### `ADDR2LINE=true|false` (PR [#774](https://github.com/pi-hole/FTL/pull/774)) {#addr2line data-toc-label='Addr2Line'}
+
+Should FTL translate its own stack addresses into code lines during the bug backtrace? This improves the analysis of crashed significantly and it is recommended to leave the option enabled.
```

Comment:
```suggestion
Should FTL translate its own stack addresses into code lines during the bug backtrace? This improves the analysis of crashed significantly. It is recommended to leave the option enabled. This option should only be disabled when `addr2line` is known to not be working correctly on the machine because, in this case, the malfunctioning `addr2line` can prevent from generating any backtrace at all.
```